### PR TITLE
Update save method to return saved data

### PR DESCRIPTION
**Pull request by tkyi**
When an object is saved/created, will return data object
Resolves 
https://github.com/screwdriver-cd/plugin-builds/issues/26 
https://github.com/screwdriver-cd/plugin-pipelines/issues/15
https://github.com/screwdriver-cd/screwdriver-datastore-imdb/issues/12

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -59,8 +59,14 @@ class MyDatastore extends Datastore {
...
     save(config, callback) {
-        imdb.set(config.table, config.params.id, config.params.data);
-        callback(null);
+        const id = config.params.id;
+        const data = config.params.data;
+
+        imdb.set(config.table, id, data);
+        callback(null, {
+            id,
+            data
```
Comment (d2lam) [inline index.js]: we don't want the key `data`

File: index.js [hunk 2]
```diff
@@ -59,8 +59,12 @@ class MyDatastore extends Datastore {
...
      */
     save(config, callback) {
-        imdb.set(config.table, config.params.id, config.params.data);
-        callback(null);
+        const id = config.params.id;
+        let data = config.params.data;
+
+        imdb.set(config.table, id, data);
+        data = hoek.merge({ id }, data, false);
+        callback(null, data);
```
Comment (d2lam) [inline index.js]: can we change the test to test this?

**Comment by d2lam** [comment]
LGTM 👍

**Comment by minz1027** [comment]
LGTM 👍

**Pull request merged**